manifests/Makefile

```diff
@@ -0,0 +1,38 @@
+ARGO_ROLLOUTS_VERSION := v0.10.2
+
+TEMP_DIR := $(shell mktemp -d)
+
+.PHONY: argo-rollouts
+argo-rollouts: argo-rollouts-crds argo-rollouts-base
+
+.PHONY: argo-rollouts-crds
+argo-rollouts-crds:
+	kustomize build "https://github.com/argoproj/argo-rollouts/manifests/crds?ref=$(ARGO_ROLLOUTS_VERSION)" > crd/argo-rollouts.yaml
+
+define ARGO_ROLLOUTS_INSTALL_KUSTOMIZATION
+bases:
+  - github.com/argoproj/argo-rollouts/manifests/base?ref=$(ARGO_ROLLOUTS_VERSION)
+  - github.com/argoproj/argo-rollouts/manifests/role?ref=$(ARGO_ROLLOUTS_VERSION)
+resources:
+  - https://raw.githubusercontent.com/argoproj/argo-rollouts/$(ARGO_ROLLOUTS_VERSION)/manifests/cluster-install/argo-rollouts-clusterrolebinding.yaml
+images:
+  - name: argoproj/argo-rollouts
+    newTag: "{{.argoRollouts.version}}"
```

Comment:
Where does this come from ?

Reply:
This will result in this line: https://github.com/flanksource/karina/pull/686/files#diff-a11ce6911899b023c2d194687227963b52ebc30e81c5916471f2342d52291800R286 then it will be populated from the `cluster-config.yml` during the install phase.
```
domain: 127.0.0.1.nip.io
name: test-cluster
....
argoRollouts:
  version: v0.10.2
```

Reply:
ah makes sense. 
We probably also want to rename  argoproj/argo-rollouts to docker.io/argoproj/argo-rollouts as we have some automation for mirroring images to local registries that relies on the fully qualified path.

Reply:
Done